### PR TITLE
del() to qdel()

### PR DESCRIPTION
**Pull request by Firecage**
This PR changes a lot of del() to qdel()

I still have to test if I accidentally broke anything, but I doubt anything broke.

PS, if anyone can see wrong qdels in the files changes, please inform me

Edit: Well, did test it, didn't see anything broken, though perhaps a maintainer should test for themselves first before they merge, just in case I missed anything important.

### CODE REVIEW

File: code/datums/gas_mixture.dm [hunk 1]
```diff
@@ -321,7 +321,7 @@ What are the archived variables for?
 				trace_gases += corresponding
 			corresponding.moles += trace_gas.moles
 
-//	del(giver)
+//	qdel(giver)
```
Comment (duncathan) [inline code/datums/gas_mixture.dm]: I'm glad that we take time to maintain commented code

File: code/controllers/subsystem/server_maintenance.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@
 				if(!istype(C.mob, /mob/dead))
 					log_access("AFK: [key_name(C)]")
 					C << "<span class='danger'>You have been inactive for more than 10 minutes and have been disconnected.</span>"
-					del(C)
+					qdel(C)
```
Comment (MrStonedOne) [inline code/controllers/subsystem/server_maintenance.dm]: this is a client. and shouldn't be qdel'ed

File: code/game/area/areas.dm [hunk 3]
```diff
@@ -391,7 +391,7 @@
 			else
 				qdel(AM)
 		if(istype(T, /turf/simulated))
-			del(T)
+			qdel(T)
```
Comment (CorruptComputer) [inline code/game/area/areas.dm]: image

File: code/modules/mob/living/carbon/human/blood.dm [hunk 4]
```diff
@@ -282,7 +282,7 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 	// Only a certain number of drips (or one large splatter) can be on a given turf.
 	for(var/obj/effect/decal/cleanable/blood/drip/drop in T)
 		drips |= drop.drips
-		del(drop)
+		qdel(drop)
```
Comment (MrStonedOne) [inline code/modules/mob/living/carbon/human/blood.dm]: Congratulations you just fixed a fuck ton of lag caused by humans bleeding.

File: code/modules/mob/login.dm [hunk 5]
```diff
@@ -30,7 +30,7 @@
 
 	client.images = null				//remove the images such as AIs being unable to see runes
 	client.screen = list()				//remove hud items just in case
-	if(hud_used)	del(hud_used)		//remove the hud objects
+	if(hud_used)	qdel(hud_used)		//remove the hud objects
```
Comment (MrStonedOne) [inline code/modules/mob/login.dm]: This will make round start have a fuck ton less lag.

File: code/game/objects/items/weapons/tanks/tanks.dm [hunk 6]
```diff
@@ -33,7 +33,7 @@
 
 /obj/item/weapon/tank/Destroy()
 	if(air_contents)
-		del(air_contents)
+		qdel(air_contents)
```
Comment (MrStonedOne) [inline code/game/objects/items/weapons/tanks/tanks.dm]: explosions should now have 1.6ds less lag.

File: code/controllers/configuration.dm [hunk 7]
```diff
@@ -576,10 +576,10 @@
 		var/datum/game_mode/M = new T()
 		//world << "DEBUG: [T], tag=[M.config_tag], prob=[probabilities[M.config_tag]]"
 		if(!(M.config_tag in modes))
-			del(M)
+			qdel(M)
 			continue
 		if(probabilities[M.config_tag]<=0)
-			del(M)
+			qdel(M)
```
Comment (MrStonedOne) [inline code/controllers/configuration.dm]: holy jesus! look at all that round start lag.

File: code/_onclick/hud/action.dm [hunk 8]
```diff
@@ -45,7 +45,7 @@
 	if(button)
 		if(T.client)
 			T.client.screen -= button
-		del(button)
+		qdel(button)
```
Comment (MrStonedOne) [inline code/_onclick/hud/action.dm]: More mob life lag fixed.

**Comment by MrStonedOne** [comment]
I have reviewed every line changed, and i see nothing wrong.

**Comment by Firecage** [comment]
Wait, so I just fixed a lot of lag? Yaaaaaaaaay. Well, Anyway, will prob update the first post later, but I did also test things. I don't see anything that I might have broken, buuuut, maybe I maintainer would want to test it for themselves before they merge it.

**Comment by Miauw62** [comment]
MSO says it's clear.

**Pull request merged**